src/redisearch.h

```diff
@@ -302,6 +303,7 @@ typedef struct RSIndexResult {
    *******************************************************************************/
   /* The docId of the result */
   t_docId docId;
+  const RSDocumentMetadata *dmd;
```

Comment:
this addition here violates the `IMPORTANT` comment above. Should we move it to another part of the struct?